frontend/src/scenes/experiments/MetricSelector.tsx

```diff
@@ -131,6 +135,15 @@ export function ExperimentInsightCreator({ insightProps }: { insightProps: Insig
                     TaxonomicFilterGroupType.Elements,
                     TaxonomicFilterGroupType.HogQLExpression,
                 ]}
+                actionsTaxonomicGroupTypes={
```

Comment:
the backend feeds into the old trends query runner class, will that handle dataWarehouse actions? Basically this line: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/trend_experiment_result.py#L225 unsure if this automagically supports these new types.

Not sure if you've tried running an experiment locally yet. We'll also add a breakdown by feature flag to the metric, so I imagine this table needs to have the flag properties as well? Unless its a funnel with the first event coming from non-data warehouse (which will automatically populate it) - but I assume that still requires a personID column in the warehouse to match the right persons?

Reply:
Oh hmm, if the backend depends on old trends query runner, this won't work yet